### PR TITLE
adding Provisions pop-up

### PR DESCRIPTION
**Pull request by TalBenAvi**

### CODE REVIEW

File: public/assets/provisions_pop-up_background.svg
Comment (igalst) [inline public/assets/provisions_pop-up_background.svg:1]: remove

File: workspaces/website/src/blocks/HomepageHero.tsx [hunk 1]
```diff
@@ -9,144 +9,158 @@ type Props = {
...
 };
 import { Intro } from "./Intro";
+import PopUp from "@ui/PopUp/PopUp";
+import { useState } from "react";
 
 export const HomepageHero = ({ seo }: Props) => {
+  const [isOpen, setIsOpen] = useState<boolean>(true);
+  const toggleModal = () => {
+    setIsOpen((prevState) => !prevState);
+  };
```
Comment (igalst) [inline workspaces/website/src/blocks/HomepageHero.tsx]:
```suggestion
  const toggleModal = () => setIsOpen((prevState) => !prevState);
```

File: workspaces/website/src/components/PopUp/PopUp-Text.svg
Comment (igalst) [inline workspaces/website/src/components/PopUp/PopUp-Text.svg:1]: Filename to lower-case

File: workspaces/website/src/components/PopUp/PopUp-background.png
Comment (igalst) [inline workspaces/website/src/components/PopUp/PopUp-background.png:1]: Filename to lower-case

File: workspaces/website/src/components/PopUp/PopUp.tsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  isOpen: boolean;
+};
+
+declare global {
+  interface Window {
+    dataLayer: {
+      event: string;
+    }[];
+  }
+}
```
Comment (igalst) [inline workspaces/website/src/components/PopUp/PopUp.tsx]: Move to where the dataLayer is added from

File: workspaces/website/src/components/PopUp/PopUp.tsx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+const PopUp = ({ toggleModal, isOpen }: Props) => {
+  const gtmEvent = (event: string) => window?.dataLayer.push({ event });
+
+  const gtmEventClickReadMore = () => gtmEvent("Provisions popup click");
+
+  const gtmEventClickClose = () => gtmEvent("Provisions popup close");
```
Comment (igalst) [inline workspaces/website/src/components/PopUp/PopUp.tsx]:
```suggestion
  const gtmEvent = (event: string) => window?.dataLayer.push({ event });
  const gtmEventClickReadMore = () => gtmEvent("Provisions popup click");
  const gtmEventClickClose = () => gtmEvent("Provisions popup close");
  
```

File: workspaces/website/src/components/PopUp/PopUp.tsx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                />
+              </g>
+              <defs>
+                <clipPath id="clip0_1_600">
+                  <rect width="28" height="28" fill="white" />
+                </clipPath>
+              </defs>
+            </svg>
```
Comment (igalst) [inline workspaces/website/src/components/PopUp/PopUp.tsx]: move to a separate file

File: workspaces/website/src/components/PopUp/PopUp.tsx [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+                  fill="currentColor"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    fill-rule="evenodd"
+                    clip-rule="evenodd"
+                    d="M12.9697 3.96967C13.2626 3.67678 13.7374 3.67678 14.0303 3.96967L21.5303 11.4697C21.671 11.6103 21.75 11.8011 21.75 12C21.75 12.1989 21.671 12.3897 21.5303 12.5303L14.0303 20.0303C13.7374 20.3232 13.2626 20.3232 12.9697 20.0303C12.6768 19.7374 12.6768 19.2626 12.9697 18.9697L19.1893 12.75H3C2.58579 12.75 2.25 12.4142 2.25 12C2.25 11.5858 2.58579 11.25 3 11.25H19.1893L12.9697 5.03033C12.6768 4.73744 12.6768 4.26256 12.9697 3.96967Z"
+                    fill="currentColor"
+                  />
+                </svg>
```
Comment (igalst) [inline workspaces/website/src/components/PopUp/PopUp.tsx]: move to a separate file

File: workspaces/website/src/components/PopUp/PopUp.tsx
Comment (igalst) [inline workspaces/website/src/components/PopUp/PopUp.tsx:1]: Rename to `ProvisionsPopup`

File: workspaces/website/src/components/PopUp/PopUp.tsx [hunk 6]
```diff
@@ -0,0 +1,152 @@
+import { Box, Button, Image, Icon, Link, Fade } from "@chakra-ui/react";
+import Background from "./PopUp-background.png";
+import Logo from "./PopUp-Text.svg";
+
+type Props = {
+  toggleModal: () => void;
+  isOpen: boolean;
+};
```
Comment (igalst) [inline workspaces/website/src/components/PopUp/PopUp.tsx]: Moe the the state and the toggle to this component

File: workspaces/website/src/components/PopUp/PopUp.tsx [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+
+declare global {
+  interface Window {
+    dataLayer: {
+      event: string;
+    }[];
+  }
+}
+
+const PopUp = ({ toggleModal, isOpen }: Props) => {
```
Comment (igalst) [inline workspaces/website/src/components/PopUp/PopUp.tsx]: Rename to ProvisionsPopup

**Review by igalst — CHANGES_REQUESTED** [review]

**Comment by HoseinNikmaramOfficial** [comment]
@TalBenAvi

**Review by igalst — APPROVED** [review]

**Pull request merged**